Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG PYTHON_VER
+FROM python:${PYTHON_VER}
+ARG PYTHON_VER
+ENV PYTHON_VER ${PYTHON_VER}
```

Comment:
the Python docker files set an environment variable `PYTHON_VERSION`. Could we use that instead?